packages/experiments/src/components/Button/SplitButton/SplitButton.test.tsx

```diff
@@ -57,4 +58,26 @@ describe('SplitButton', () => {
     const tree = component.toJSON();
     expect(tree).toMatchSnapshot;
   });
+
+  it('focuses correctly when focus is triggered via ISplitButton interface', () => {
+    const wrapper = mount(
+      <div>
+        <SplitButton content="Split Button 1" />
+        <SplitButton content="Split Button 2" />
+        <SplitButton content="Split Button 3" />
+      </div>
+    );
+
+    const buttons = wrapper.getDOMNode().querySelectorAll('span.ms-SplitButton > button.ms-Button') as NodeListOf<HTMLButtonElement>;
```

Comment:
Nit -- You can also use [wrapper.find](https://airbnb.io/enzyme/docs/api/ReactWrapper/find.html) and since the look-up is relative to `wrapper` you don't need as complicated a selector. I think.